masq_lib/src/shared_schema.rs

```diff
@@ -526,9 +526,110 @@ impl ConfiguratorError {
+     zero-hop means that your Node will operate as its own MASQ Network and will not communicate with any \
+     other Nodes. --ip, --neighbors, and --clandestine-port are incompatible with --neighborhood_mode \
+     zero-hop.\n\n\
+     originate-only means that your Node will not accept connections from any other Node; it \
+     will only originate connections to other Nodes. This will reduce your Node's opportunity to route \
+     data (it will only ever have two neighbors, so the number of routes it can participate in is limited), \
+     it will reduce redundancy in the MASQ Network, and it will prevent your Node from acting as \
+     a connection point for other Nodes to get on the Network; but it will enable your Node to operate in \
+     an environment where your network hookup is preventing you from accepting connections, and it means \
+     that you don't have to forward any incoming ports through your router. --ip and --clandestine_port \
+     are incompatible with --neighborhood_mode originate-only.\n\n\
+     consume-only means that your Node will not accept connections from or route data for any other Node; \
+     it will only consume services from the MASQ Network. This mode is appropriate for devices that \
+     cannot maintain a constant IP address or stay constantly on the Network. --ip and --clandestine_port \
+     are incompatible with --neighborhood_mode consume-only.\n\n\
+     standard means that your Node will operate fully unconstrained, both originating and accepting \
+     connections, both consuming and providing services, and when you operate behind a router, it \
+     requires that you forward your clandestine port through that router to your Node's machine.");
+        assert_eq!(MAPPING_PROTOCOL_HELP, "The Node can speak three protocols to your router to make it allow outside Nodes to connect inward \
+    through it to your machine. These three protocols are pcp, pmp, and igdp. The Node can try them one \
+    by one to determine which your router supports, but if you happen to know already, you can supply the \
+    name of the protocol here. If you've taken care of port mapping in some other way, \
+    and you don't need Node to negotiate with your router, say 'none' here and be sure to specify your \
+    public IP address with the --ip parameter. If the Node communicates successfully with your router, \
+    it will remember the protocol it used, and on its next run it will try that protocol first, unless \
+    you specify a different protocol on the command line.");
+        assert_eq!(REAL_USER_HELP,"The user whose identity Node will assume when dropping privileges after bootstrapping. Since Node refuses to \
+     run with root privilege after bootstrapping, you might want to use this if you start the Node as root, or if \
+     you start the Node using pkexec or some other method that doesn't populate the SUDO_xxx variables. Use a value \
+     like <uid>:<gid>:<home directory>.");
+    }
+
```

Comment:
There are some files...I cannot point to them directly because the code is so far unchanged but it occurred to me that you should also take care of `static` global variables.

In a case like this:
```
lazy_static! {
    static ref UI_PORT_HELP: String = format!(
        "If the Daemon is listening for connections at some port other than {}, specify that port \
         here. Must be between {} and {}.",
        DEFAULT_UI_PORT, LOWEST_USABLE_INSECURE_PORT, HIGHEST_USABLE_PORT
    );
    static ref DEFAULT_UI_PORT_STRING: String = format!("{}", DEFAULT_UI_PORT);
}
```

Maybe not a good example, here you can fix from the second one to:
static ref DEFAULT_UI_PORT_STRING: String = DEFAULT_UI_PORT.to_string();

The first assertion could look like this: 
```   
    assert_eq!(UI_PORT_HELP, format!(
        "If the Daemon is listening for connections at some port other than {}, specify that port \
         here. Must be between {} and {}.",
        DEFAULT_UI_PORT, LOWEST_USABLE_INSECURE_PORT, HIGHEST_USABLE_PORT
    );
``` 
    
 Put differently, should look exactly like the real declaration. :)